SpatialDilatedMaxPooling.lua

```diff
@@ -0,0 +1,72 @@
+local THNN = require 'nn.THNN'
+local SpatialDilatedMaxPooling, parent = torch.class('nn.SpatialDilatedMaxPooling', 'nn.SpatialMaxPooling')
+
+function SpatialDilatedMaxPooling:__init(kW, kH, dW, dH, padW, padH, dilationW, dilationH)
+   parent.__init(self, kW, kH, dW, dH, padW, padH)
+
+   self.dilationW = dilationW or 1
+   self.dilationH = dilationH or 1
+end
+
+function SpatialDilatedMaxPooling:ceil()
```

Comment:
yes, but since you are deriving nn.SpatialDilatedMaxPooling from nn.SpatialMaxPooling, the methods are defined already.